App/App.xcodeproj/project.pbxproj

```diff
@@ -422,7 +421,7 @@
 				CURRENT_PROJECT_VERSION = 1;
 				DEAD_CODE_STRIPPING = YES;
 				DEVELOPMENT_ASSET_PATHS = "\"App/Preview Content\"";
-				DEVELOPMENT_TEAM = 9PC9DZ9559;
+				DEVELOPMENT_TEAM = 5QJLL8HM6C;
```

Comment:
Please not modify development target since we need to submit with own team.

Reply:
Fixed on the latest commit